Lib/test/test_codecs.py

```diff
@@ -3188,11 +3191,13 @@ def test_incremental(self):
         self.assertEqual(decoded, ('abc', 3))
 
     def test_mbcs_alias(self):
-        # Check that looking up our 'default' codepage will return
-        # mbcs when we don't have a more specific one available
-        with mock.patch('_winapi.GetACP', return_value=123):
-            codec = codecs.lookup('cp123')
-            self.assertEqual(codec.name, 'mbcs')
+        # On Windows, the encoding name must be the ANSI code page
+        encoding = locale.getpreferredencoding(False)
+        self.assertTrue(encoding.startswith('cp'), encoding)
```

Comment:
This will fail if `PYTHONUTF8` is set in the environment, because it overrides `getpreferredencoding(False)` and `_get_locale_encoding()`.